webknossos/Changelog.md

```diff
@@ -7,14 +7,21 @@ and this project adheres to [Semantic Versioning](http://semver.org/) `MAJOR.MIN
 For upgrade instructions, please check the respective *Breaking Changes* sections.
 
 ## Unreleased
-[Commits](https://github.com/scalableminds/webknossos-libs/compare/v0.8.25...HEAD)
+[Commits](https://github.com/scalableminds/webknossos-libs/compare/v0.8.27...HEAD)
 
 ### Breaking Changes
 
 ### Added
 
 ### Changed
 
+### Fixed
+
+
+## [0.8.27](https://github.com/scalableminds/webknossos-libs/releases/tag/v0.8.27) - 2021-12-09
```

Comment:
I had to skip 0.8.26, since I already created that release and thought that the new auto-release functionality would do the rest.